pkg/arvo/gen/hood/merge/help.txt

```diff
@@ -0,0 +1,110 @@
+destination desk.
+                                                                      
+%only-that: create a merge commit with exactly the contents of the
+source commit.
+                                                                      
+%take-this: create a merge commit with exactly the contents of the
+destination desk except take any files from the source commit which
+are not in the destination desk.
+                                                                      
+%take-that: create a merge commit with exactly the contents of the
+source commit except preserve any files from the destination desk
+which are not in the source commit.
+                                                                      
+%meet-this: merge as in %meet, except if both sides changed the same
+file, use the version in the destination desk.
+                                                                      
+%meet-that: merge as in %meet, except if both sides changed the same
+file, use the version in the source commit.
+                                                                      
+# Examples and notes:
+                                                                      
+The most common merge strategy is %mate, which is a normal 3-way
+merge which aborts on conflict.
+                                                                      
+%take-that is useful to "force" an OTA.  After running %take-that,
+you're guaranteed to have exactly the files in the source commit plus
+any files you separately added.
+                                                                      
+We speak of merging into a destination *desk* from a source *commit*
+becaus while you can only merge on top of a desk, you can merge from
+historical commits.  For example,
+                                                                      
+    |merge %old our %home, =cas ud+5, =gem %init
+                                                                      
+will create a new desk called %old with the 5th commit in %home.
+You can revert the contents of a desk to what they were yesterday
+with
+                                                                      
+    |merge %home our %home, =cas da+(sub now ~d1), =gem %only-that
+                                                                      
+Note this is a normal %only-that merge, which means you're creating a
+*new* commit with the old *contents*.
+                                                                      
+%meld is rarely used on its own, however if you specify %auto or
+omit the merge strategy, %kiln will run a %meld merge into a scratch
+desk and then annotate the conflicts there.
+                                                                      
+If you resolve merge conflicts manually, for example by mounting the
+desks, copying the files in unix and then running |commit, you
+should usually run an %only-this merge.  This will not change the
```

Comment:
As a note for the future, I think we should eventually automate this workflow for development.  It could be related to the `-work` thread.

Reply:
Good idea.  Would love it to to just pop up your `$EDITOR`, but that's probably a bridge too far...